.github/workflows/artifacts-updater.yml

```diff
@@ -24,7 +24,7 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  build-artifacs:
+  build-artifacts:
```

Comment:
This should be _mostly_ harmless, but could break required checks

Reply:
this is fine